src/java/main/org/apache/zookeeper/Login.java

```diff
@@ -45,6 +47,7 @@
 import java.util.Set;
 
 public class Login {
+    private static final String KINIT_COMMAND_DEFAULT = "/usr/bin/kinit";
```

Comment:
Got it. In fact I was not advocating the replacement of KINIT_COMMAND_DEFAULT by KINIT_COMMAND, but just move the constant to ZKConfig in addition to KINIT_COMMAND. But 2) answered my question.